library/Icingadb/Widget/ItemList/BaseDowntimeListItem.php

```diff
@@ -131,24 +132,33 @@ protected function assembleTitle(BaseHtmlElement $title)
             $link = $this->createServiceLink($this->item->service, $this->item->service->host, true);
         }
 
+        if ($this->item->is_flexible) {
+            if ($link !== null) {
+                $template = t('{{#link}}Flexible Downtime{{/link}} for %s');
+            } else {
+                $template = t('{{#link}}Flexible Downtime{{/link}}');
```

Comment:
If we don't require a template string, we shouldn't use one. It only makes the translation more difficult. Every translatable string that is not used anywhere else must be explicitly translated. Even a single difference like a comma or so leads to this. Here, the `{{#link}}` and `{{/link}}` placeholders are redundant because there's no need for them. They are only required due to the addition of the `for`, which is missing here.

So what I would like to see instead, is that in case of no object link, the subject is constructed the same as previously. Which leads to e.g. `t('Fixed Downtime')` again and we have less redundancy in the translation message catalogs.